infra/aws/terraform/cdn.packages.k8s.io/main.tf

```diff
@@ -15,22 +15,22 @@ limitations under the License.
 */
 
 terraform {
-  backend "s3" {
-    bucket       = "cdn-packages-k8s-io-tfstate"
-    key          = "terraform.tfstate"
-    region       = "eu-central-1"
-    role_arn     = "arn:aws:iam::309501585971:role/Provisioner"
-    session_name = "cdn-packages-k8s-io-terraform"
-  }
-
-  required_version = "~> 1.5.0"
+  required_version = "~> 1.1"
```

Comment:
Why is version downgraded to 1.1?

Reply:
because there aren't any features used that are only available in 1.1+ 

for example - if we started utilizing the new Terraform native test functionality, we would need to change this to 
```hcl
  required_version = "~> 1.6"
```

Its not like a runtime or programming language where there are performance improvements, later versions mean newer features and you typically raise the MSV in response to the features utilized

Reply:
@bryantbiggs I verified that using `~> 1.1` allows using newer Terraform versions (e.g. 1.5) so this can stay IMO